napari/utils/notebook_display.py

```diff
@@ -86,7 +87,13 @@ def _clean_alt_text(self, alt_text):
                 str(alt_text)
             )  # cleaner won't recognize unescaped script tags
             cleaner = Cleaner()
-            doc = document_fromstring(alt_text)
+            try:
+                doc = document_fromstring(alt_text)
+            except ParserError:
+                alt_text = ""
+                warn(
+                    'The provided alt text does not constitute valid html, so it was discarded.'
```

Comment:
Maybe we want to warn with a specific warning class, so that it can be filtered/turned into error. If we don't I believe the default class will be USerWarning. I guess we could even warng with `ParseError`... even if that may be weird.